nano/test_common/testutil.hpp

```diff
@@ -56,13 +56,15 @@
 	}
 
 /** Expects that the condition becomes true within the deadline */
-#define EXPECT_TIMELY(time, condition)             \
-	system.deadline_set (time);                    \
-	std::error_code ec;                            \
-	while (!(condition) && !(ec = system.poll ())) \
-	{                                              \
-	}                                              \
-	EXPECT_NO_ERROR (ec);
+#define EXPECT_TIMELY(time, condition)                  \
+	system.deadline_set (time);                         \
+	{                                                   \
```

Comment:
Whats's the reason for the extra scope here?

Reply:
It is a C style macro that creates a local variable.
It is not ideal, so I limited its scope and added an underscore to the variable name.